.circleci/config.yml

```diff
@@ -1,9 +1,75 @@
+orbs:
+  # The Windows orb gives us everything we need to start using the Windows executor.
+  win: circleci/windows@2.4.0
+
 defaults: &defaults
   docker:
     - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:go1.16-tf1.0-tg0.30
 
-version: 2
+version: 2.1
 jobs:
+  test_windows:
+    executor:
+      name: win/default
+      size: "large"
```

Comment:
Not sure yet if this is necessary - it hasn't seemed to make much difference in the tests hanging.